db/migrate/20240829142639_remove_unique_constraint_from_tag_users_indexes.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class RemoveUniqueConstraintFromTagUsersIndexes < ActiveRecord::Migration[7.1]
```

Comment:
Just realized this might need `disable_ddl_transaction!` because of `CONCURRENTLY`.